dashboard/app/controllers/api/v1/peer_review_submissions_controller.rb

```diff
@@ -0,0 +1,37 @@
+class Api::V1::PeerReviewSubmissionsController < ApplicationController
+  authorize_resource class: :peer_review_submissions
+  # GET /api/v1/peer_review_submissions/index_escalated
+  def index_escalated
+    # Get a list of submissions that have been escalated
+    # This will return an array of objects containing
+    # - Submitter
+    # - Course name
+    # - Unit
+    # - Level name
+    # - Submission date
+    # - Date it was escalated
+    # - ID of review
+
+    escalated_submissions = Hash.new
+
+    escalated_reviews = PeerReview.escalated.limit(50)
+
+    escalated_reviews.each do |review|
+      plc_course_unit = review.script.plc_course_unit
+
+      escalated_submissions[review.user_level.id] = {
+        submitter: review.submitter.name,
+        course_name: plc_course_unit.plc_course.name,
+        unit_name: plc_course_unit.name,
+        level_name: review.level.name,
+        submission_date: review.created_at,
+        escalated_date: review.updated_at,
+        review_id: review.id
+      }
+    end
```

Comment:
Should this be handled in a Serializer?
Then we could do something like

```ruby
render json: escalated_reviews, each_serializer: Api::V1::PeerReviewSubmissionSerializer
```

Reply:
I don't think a serializer is appropriate - we are going from n peer reviews to somewhere between .5n and n submissions. The reason we're doing the hash map is so if a submission has two escalated reviews, it only occurs once it what's returned.

Reply:
Oh that makes sense. I didn't notice the deduping